data/mozilla_deepspeech_wav2txt.py

```diff
@@ -36,10 +36,10 @@
 except ImportError:
     from pipes import quote
 
-_global_modules_needed_by_name = ["deepspeech-gpu==0.6.1"]
+_global_modules_needed_by_name = ["deepspeech-gpu==0.10.0a3"]
```

Comment:
The latest version is `0.9.3`, and `0.10.0a3` does not exist on PyPI.
https://pypi.org/project/deepspeech-gpu/

Reply:
`0.10.0a3` is a pre-release https://pypi.org/project/deepspeech-gpu/0.10.0a3/

Reply:
Thanks, I see it now.